Sources/Commands/Describe.swift

```diff
@@ -62,15 +62,19 @@ fileprivate struct DescribedPackage: Encodable {
         self.dependencies = package.manifest.dependencies.map { DescribedPackageDependency(from: $0) }
         self.defaultLocalization = package.manifest.defaultLocalization
         self.platforms = package.manifest.platforms.map { DescribedPlatformRestriction(from: $0) }
-        self.products = package.products.map {
+        // SwiftPM considers tests to be products, which is not how things are presented in the manifest.
+        let nonTestProducts = package.products.filter{ $0.type != .test }
+        self.products = nonTestProducts.filter{ $0.type != .test }.map {
```

Comment:
second filter redundant?

Reply:
Ah yes, good catch — I factored some code but forgot this one.  Will adjust.